README.md

```diff
@@ -2,6 +2,29 @@
 
 Utility to vectorize raster images :rocket:
 
+## Local development
+
+1. Install system dependencies for pypotrace:
+
+	```bash
+	sudo apt-get update
+	sudo apt-get install build-essential python-dev libagg-dev libpotrace-dev pkg-config
+	```
+
+2. Create the conda development environment and activate it
+
+	```bash
+	conda env create -n dev -f envs/dev.yaml
```

Comment:
I would probably include how to install anaconda, or a link to e.g. https://medium.com/ayuth/install-anaconda-on-macos-with-homebrew-c94437d63a37

Reply:
Very good point, I added this in the readme documentation - and also a dev container that has already all the tools like conda already installed, for faster development